bin/MadGraph5_aMCatNLO/cards/production/13TeV/diphotonJets_5f_NLO_FXFX_Mgg80ToInf/diphotonJets_5f_NLO_FXFX_Mgg80ToInf_cuts.f

```diff
@@ -372,13 +379,39 @@ logical function passcuts_user(p,istatus,ipdg)
 c End photon isolation
       endif
 
+C
+C     PDG SPECIFIC CUTS (PT/M_IJ)
+C
+      do i=nincoming+1,nexternal-1
+         if(etmin(i).gt.0d0 .or. etmax(i).gt.0d0)then
+            tmpvar = pt_04(p(0,i))
+            if (tmpvar.lt.etmin(i)) then
+               passcuts_user=.false.
+               return
+            elseif (tmpvar.gt.etmax(i) .and. etmax(i).gt.0d0) then
+               passcuts_user=.false.
+               return
+            endif
+         endif
+         do j=i+1, nexternal-1
+            if (mxxmin(i,j).gt.0d0)then
+               if (invm2_04(p(0,i),p(0,j),1d0).lt.mxxmin(i,j)**2)then
+                  passcuts_user=.false.
+                  return
+               endif
```

Comment:
which cut is this?

Reply:
@agrohsje  @sihyunjeon 
I do not know exactly what cut is this.  Since I could not run the old cards with new MG2.6 version, I had to create the cuts.f file with new mg2.6 and add the dip photon selection.  Could you please suggest whom should I contact in the generator group for this clarification. 

@sihyunjeon ,  about the Eta Gamma, I changed NLO cards to 3.5= etagamma to be consistent with LO cards

Reply:
The right people are already reviewing. :-) Now I see. You moved to 2.6 and these are the new cuts based on pdgids that were not available before. So in principle now you can implement at card level a cut on tops, this is the corresponding fortran part. I iniititally assumed you wrote this.

Reply:
Thanks,  Yes, I shifted to MG2.6.